packages/beeq/src/components/date-picker/_storybook/bq-date-picker.stories.tsx

```diff
@@ -0,0 +1,309 @@
+          You can provide more context detail by adding a tooltip to the label.
+        </bq-tooltip>
+      `
+    : nothing;
+  const labelTemplate = html`
+    <label class="flex flex-grow items-center" slot=${ifDefined(!args.optionalLabel ? 'label' : null)}>
+      Date picker label ${tooltipTemplate}
+    </label>
+  `;
+  const label = !args.optionalLabel
+    ? labelTemplate
+    : html`
+        <div slot="label" class="flex flex-1">
+          ${labelTemplate}
+          <span class="text-text-secondary">Optional</span>
+        </div>
+      `;
+  /**
+   * * Converts a Date object to an ISO 8601 string representation.
+   * This function is used only for demonstration purposes in Storybook.
+   */
+
+  const dateToIsoString = (date: Date): string => {
+    return date.toISOString().split('T')[0];
+  };
+
+  const isDate = (date: Date): boolean => {
+    if (!args.customDisallowedDate) {
+      return false;
+    }
+
+    const dateString = dateToIsoString(date);
+    return args.customDisallowedDate.split(',').some((date: string) => date.trim() === dateString);
+  };
+
+  return html`
+    <div class="w-[280px]">
+      <bq-date-picker
+        ?autofocus=${args.autofocus}
+        clear-button-label=${args['clear-button-label']}
+        distance=${args.distance}
+        ?disable-clear=${args['disable-clear']}
+        ?disabled=${args.disabled}
+        form=${ifDefined(args.form)}
+        min=${ifDefined(args.min)}
+        max=${ifDefined(args.max)}
+        locale=${ifDefined(args.locale)}
+        ?show-outside-days=${args['show-outside-days']}
+        first-day-of-week=${args['first-day-of-week']}
+        name=${ifDefined(args.name)}
+        ?open=${args.open}
+        panel-height=${args['panel-height']}
+        placeholder=${args.placeholder}
+        placement=${args.placement}
+        ?required=${args.required}
+        skidding=${args.skidding}
+        strategy=${args.strategy}
+        validation-status=${args['validation-status']}
+        type=${args.type}
+        months=${args.months}
+        value=${ifDefined(args.value)}
+        .format-options=${args['format-options']}
+        .isDateDisallowed=${isDate}
+        @bqBlur=${args.bqBlur}
+        @bqChange=${args.bqChange}
+        @bqClear=${args.bqClear}
+        @bqFocus=${args.bqFocus}
+      >
+        ${!args.noLabel ? label : nothing}
+        ${args.prefix ? html`<bq-icon name="user-circle" slot="prefix"></bq-icon>` : nothing}
+        ${args.suffix ? html`<bq-icon name="arrow-down" slot="suffix"></bq-icon>` : nothing}
+      </bq-date-picker>
+    </div>
+  `;
+};
+
+export const Default: Story = {
+  render: Template,
+};
+
+export const Range: Story = {
+  render: Template,
+  args: {
+    type: 'range',
+    months: 2,
+  },
+};
+
+export const Multi: Story = {
+  render: Template,
+  args: {
+    type: 'multi',
+    months: 2,
+  },
+};
+
+export const InitialValue: Story = {
+  render: (args) => html`
+    <div class="flex flex-row gap-4">
```

Comment:
You can use a grid instead:
```suggestion
    <div class="grid grid-cols-1 gap-m sm:grid-cols-3">
```
this will guarantee a responsive layout.